python/torch_mlir/csrc/base_lazy_backend/shape_inference.cpp

```diff
@@ -49,5 +49,12 @@ std::vector<torch::lazy::Shape> compute_shape_where(
   return {Shape(self.scalar_type(), self.sizes().vec())};
 }
 
+std::vector<torch::lazy::Shape> compute_shape_bucketize(
```

Comment:
@antoniojkim, can you review the changes in this file?

Reply:
changes in this file look good to me